From 4fe5073b0220df48a0ead9118745c79a0e1eee2b Mon Sep 17 00:00:00 2001
From: Sergey 'Jin' Bostandzhyan <jin@mediatomb.cc>
Date: Mon, 11 May 2020 17:49:27 +0200
Subject: [PATCH 2/3] Assume hub-ctrl is installed on the system and is in
 $PATH

---
 klippy/chelper/__init__.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/klippy/chelper/__init__.py b/klippy/chelper/__init__.py
index 14e115f8..63f1ba07 100644
--- a/klippy/chelper/__init__.py
+++ b/klippy/chelper/__init__.py
@@ -192,7 +192,7 @@ HC_COMPILE_CMD = "gcc -Wall -g -O2 -o %s %s -lusb"
 HC_SOURCE_FILES = ['hub-ctrl.c']
 HC_SOURCE_DIR = '../../lib/hub-ctrl'
 HC_TARGET = "hub-ctrl"
-HC_CMD = "sudo %s/hub-ctrl -h 0 -P 2 -p %d"
+HC_CMD = "hub-ctrl -h 0 -P 2 -p %d"
 
 def run_hub_ctrl(enable_power):
     srcdir = os.path.dirname(os.path.realpath(__file__))
-- 
2.25.4

